src/autoDeleteMessages.ts

```diff
@@ -1,23 +1,53 @@
 import { BotContext } from './context';
 
-export interface AutoDeleteMessage {
-  [messageId: number]: number;
-}
+const expired = (createdAt: number, timeout: number) => {
+  const minute = 60000;
+  return Math.floor((Date.now() - createdAt) / minute) >= timeout;
+};
```

Comment:
I isolated your expression evaluation into this `expired` function, which makes it a bit more understandable

Reply:
wayyy better!